frontend/packages/ux-editor/src/components/FormComponent/FormComponent.tsx

```diff
@@ -121,7 +121,9 @@ export const FormComponent = memo(function FormComponent({
           ) : (
             <div className={classes.formComponentTitle}>
               <span className={classes.icon}>
-                {Icon && <Icon title={getComponentTitleByComponentType(component.type, t)} />}
+                {Icon && (
+                  <Icon title={getComponentTitleByComponentType(component.type, t)} aria-hidden />
```

Comment:
this icons should be ignored by the screen readers, hence I added aria-hidden on them.